actions/router.delete.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 description: Delete Router
 enabled: true
```

Comment:
As this isn't implemented you might want to set this and router.update to false.